woodstock-external/prototype/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
Remove this file. It's not used. The JAR plugin generates the file with these contents, if built on Java 21:

```
Manifest-Version: 1.0
Created-By: Maven JAR Plugin 3.4.1
Build-Jdk-Spec: 21
```